<launch>
    <node pkg="arx_r5_controller" type="R5Controller" name="arm">
        <param name="arm_control_type" value="normal"/>
        <param name="arm_can_id" value="can1"/>
        <param name="arm_pub_topic_name" value="/arm_status"/>
        <param name="arm_sub_topic_name" value="/arm_cmd"/>
        <param name="arm_end_type" value="0"/>
    </node>
</launch>
